documentation/source/users/rmg/liquids.rst

```diff
@@ -372,6 +440,14 @@ and a correlation for the viscosity using parameters :math:`A, B, C, D, E`:
 .. math:: \ln \eta = A + \frac{B}{T} + C\log T + DT^E
     :label: viscosity
        
+The viscosity parameters (`A, B, C, D, E`) are provided for 150 pure solvents in RMG's database, which can be found under
+(1) `the RMG-database git repository <https://github.com/ReactionMechanismGenerator/RMG-database/blob/master/input/solvation/libraries/solvent.py>`_
+or (2) `the RMG website <https://rmg.mit.edu/database/solvation/libraries/solvent/>`_. If RMG-database is installed,
+these viscosity parameters can be found found in ``RMG-database/input/solvation/libraries/solvent.py``.
+
+You can also check whether a compound of your interest has the viscosity parameters in RMG-database by using a
```

Comment:
"of your interest has the" => "of interest has"

Reply:
it's fixed in this fixup commit 2d234db.